src/diffusers/pipelines/shap_e/pipeline_shap_e.py

```diff
@@ -130,24 +130,6 @@ def prepare_latents(self, shape, dtype, device, generator, latents, scheduler):
         latents = latents * scheduler.init_noise_sigma
```

Comment:
Seems like we're missing out on using `_execution_device` for like we did [here](https://github.com/huggingface/diffusers/pull/4114/files#diff-285029b50e12008100200a7309461d7b942969224014eede778ed7771e2aa06bR149-R152): 

https://github.com/huggingface/diffusers/blob/8b18cd8e7f6be0cf2904dfec4285d4ba98c5586f/src/diffusers/pipelines/shap_e/pipeline_shap_e.py#L124

(applies to the img2img SHAP-E pipeline too)

Am I missing out on something?

Reply:
Think this is out of scope for this PR

Reply:
Why? For some of the other pipelines, we are correctly setting the `_execution_device` like o showed above. Why leave it?

Reply:
It didn't cause an error when running the test locally, but changing now as it takes just a second to do so

Reply:
We already use `self._execution_device` in Shape-E see: https://github.com/huggingface/diffusers/blob/8b18cd8e7f6be0cf2904dfec4285d4ba98c5586f/src/diffusers/pipelines/shap_e/pipeline_shap_e.py#L305 

(this is also probably why there was no error) - so I don't think we need to change here anything

Reply:
Thanks for explaining!